File: 13-线性排序/LinearSort/ViewController.m
//
//  ViewController.m
//  LinearSort
//
//  Created by 段雨田 on 2019/11/12.
//  Copyright © 2019 段雨田. All rights reserved.
//

#import "ViewController.h"
#include "BucketSort.h"
#include "CountingSort.h"
#include "RadixSort.h"


@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
  [super viewDidLoad];
  
  
  setBucket();
  printf("\n");
  setCount();
  printf("\n");
  setRadix();

  
}


@end

File: 链表/链表/ViewController.swift
//
//  ViewController.swift
//  链表
//  https://www.cnblogs.com/rayshen/p/5145861.html
//  Created by 段雨田 on 2020/5/29.
//  Copyright © 2020 段雨田. All rights reserved.
//

import Cocoa

class ViewController: NSViewController {

  override func viewDidLoad() {
    super.viewDidLoad()

    // Do any additional setup after loading the view.
  }

  override var representedObject: Any? {
    didSet {
    // Update the view, if already loaded.
    }
  }

  @IBAction func magician(_ sender: Any) {
    
     let svc:MagicianVC = (storyboard?.instantiateController(identifier: "magicianVC"))! as MagicianVC
     self.presentAsModalWindow(svc);
    
  }
  
}

